fxevent/event.go

```diff
@@ -103,8 +109,8 @@ type Rollback struct {
 	Err      error
 }
 
-// CustomLogger is emitted whenever a custom logger is set or produces an error.
-type CustomLogger struct {
-	Function interface{}
-	Err      error
+// LoggerInitialized is emitted whenever a custom logger is set or produces an error.
```

Comment:
The latter bit of this comment sounds a little confusing since it sounds like it can be emitted when the custom logger produces an error (i.e. after initialization as well). Can we reword it such that it's emitted when an error is produced only during initialization of the custom logger?

Reply:
Synced offline. Added constructor to case where we emit LoggerInitialized due to error. Today, it's emitted in either case as documented.